src/odemis/gui/cont/tabs/fastem_main_tab.py

```diff
@@ -175,6 +173,10 @@ def on_pnl_tabs_size(self, _):
 
     def _toggle_user_settings_panel(self, _):
         self.panel.pnl_user_settings.Show(not self.panel.pnl_user_settings.IsShown())
```

Comment:
Instead of calling `IsShown()` twice, could you write first just `shown = not self.panel.pnl_user_settings.IsShown()`. Then it'll be easier to read *and* faster.